backend/src/user/endpoints.ts

```diff
@@ -58,8 +58,9 @@ export const authSuccess = (req: Request, res: Response) => {
         "window.close()" +
         "}" +
         "</script>";
-
-    res.send(script);
+    req.login(data.user, () => {
```

Comment:
Daj znać dlaczego to zmieniłeś tutaj. Nie mówię, że źle, chcę tylko się dowiedzieć, co było powodem i co to zmienia.

Reply:
To tworzy sesję. Bez użycia tego dane nie przechodzą nigdzie dalej po zalogowaniu i np. przy tworzeniu projektu autor to '', tak samo nie można wyświetlić projektu.

Reply:
To jest dobrze, ja sam to podpowiedziałem bo inaczej user nie jest dostępny w innych endpointach. Analogicznie do logout w endpointcie do wylogowania tutaj też powinno być login.